src/components/Algorithm.jsx

```diff
@@ -99,7 +99,7 @@ export const Algorithm = props => {
 	const unapprovedClass = props.data.is_accepted ? "" : " unapproved";
 	const approveButton = props.data.is_accepted ? (
```

Comment:
Personally, I find chained ?: expressions hard to read, but I might be an outlier.